symfony/plugins/orangehrmRecruitmentPlugin/config/RecruitmentPluginConfiguration.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
```

Comment:
copyright header & new line end of the file

Reply:
done